common/src/main/res/values-it/strings.xml

```diff
@@ -1339,11 +1339,11 @@ Code: {ERROR_CODE}"</string>
 
     <!-- 👍 -->
     <!-- text for info pop-up in wallet app describing 2G OK state -->
-    <string name="wallet_check_mode_info_2g_ok_text">"Per l’accesso a strutture e a manifestazioni in cui si applica la regola 2G sono ammessi certificati COVID per persone vaccinate o guarite. I certificati COVID per persone testate negative e i certificati light non sono accettati."</string>
+    <string name="wallet_check_mode_info_2g_ok_text">"L’accesso  a strutture e manifestazioni è consentito alle persone vaccinate o guarite."</string>
```

Comment:
Is there a reason for the additional space here?

Reply:
No, will be fixed in the next release. Thanks for mentioning it.